webcompat/static/css/development/components/report-form.css

```diff
@@ -15,6 +15,19 @@
     text-align: center;
     justify-content: center;
     flex-wrap: wrap;
+    margin-top: 1em;
+  }
+
+  .wc-ReportForm .wc-UploadForm {
+    border: dashed 1px rgb(204, 204, 204);
+    border-radius: 20px;
+  }
+
+  .wc-ReportForm .r-Grid-cell {
+    margin-bottom: 1em;
+  }
+  .wc-ReportForm div.r-minM--1of2:nth-child(2) .wc-Form-group {
```

Comment:
@karlcow two `class` in same div is too bad for me, same side effect than a selector.

```html
div class="wc-ReprortForm r-Grid-cell"></div>
```

For this example, a new class modifier for `r-Grid` is too specfic. So it's why I think it's better to add another wrapper.

But Maybe I can simplfiy the HTML.